### ISSUE TITLE
[DOCS] Improve the Readme to explain how to create the components

### ISSUE THREAD

**Issue by Dheyson** [CofferHub/react-atomic-tailwind-storybook#29]
## Expected Behavior

It should explain how to create the components

## Actual Behavior

It has no good explanation.

**Comment by ampsteric** [comment]
can you mention the steps and I can convert them into a nice README?
I would love to work on this. assign it to me please

**Comment by Dheyson** [comment]
Hey @ampsteric, thanks for the reply, sure!

We used to create four files for each component, based on the folder component name:
Ex:
**Button**
- index.txs
- styles.ts
- index.stories.tsx
- index.test.ts

You could structure the a new section on the readme with a title: How to Create The Components? 
We would love to see a kind of table with 2 columns with:
- File Name
- File Description

And the rows based on each file. Which I describe on the top of this comment.

That`s it, we are open to any suggestions!

Thanks again for the reply :)

**Comment by Dheyson** [comment]
Hey @ampsteric, are you still gonna work on this issue?

**Comment by ampsteric** [comment]
@Dheyson do I need to provide the description of all the files in the table?

**Comment by Dheyson** [comment]
Yes, would be great. You could link each with a component example in the repository. To show the file structure.
There is a folder called generators, that is for generate template files and shows the main file structure.
